fastlane/lib/fastlane/actions/onesignal.rb

```diff
@@ -92,6 +94,12 @@ def self.available_options
                                        sensitive: true,
                                        optional: true),
 
+          FastlaneCore::ConfigItem.new(key: :android_gcm_sender_id,
+                                       env_name: "android_gcm_sender_id",
```

Comment:
This should be `ANDROID_GCM_SENDER_ID` 🙃

Reply:
@joshdholtz My bad 😞 , it's fixed now!